automatic/1password/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,28 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$packageName = '1Password'
+
+[array]$key = Get-UninstallRegistryKey -SoftwareName '1Password*'
+
+if ($key.Count -eq 1) {
+  $key | ForEach-Object {
+    $packageArgs = @{
+      packageName    = $packageName
+      fileType       = 'EXE'
+      silentArgs     = @{"1password4" = '/SILENT'; "1password" = 'uninstall' }[$env:ChocolateyPackageName]
```

Comment:
These should not be added here.

If there is a need for different silent arguments, then that should be updated appropriately in the `update.ps1` script instead.

Reply:
Thanks for the feedback. I changed the silentArgs to only the necessary argument for the '1password' package.

Reply:
Sounds good, I have added the issue https://github.com/chocolatey-community/chocolatey-packages/issues/1771 for continued work to get the uninstall script to work on the 1password4 package as well.